content/en/docs/concepts/security/index.md

```diff
@@ -630,333 +631,277 @@ runtime. When a request comes to the proxy, the authorization engine evaluates
-conditions to specify a set of accounts assigned to this role. It contains a
-`key` and its allowed *values*. The supported `key` values of a constraint
-are listed in the
-[constraints and properties page](/docs/reference/config/authorization/constraints-and-properties/).
-
-The following example shows a `ServiceRoleBinding` named
-`test-binding-products`, which binds two subjects to the `ServiceRole` named
-`"product-viewer"` and has the following `subjects`
-
-- A service account representing service **a**, `"service-account-a"`.
-- A service account representing the Ingress service
-  `"istio-ingress-service-account"` **and** where the JWT `email` claim is
-  `"a@foo.com"`.
+The example below shows a simple policy `deny-all` which denies access to all workloads
+in the `admin` namespace.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRoleBinding
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: test-binding-products
-  namespace: default
+  name: deny-all
+  namespace: admin
 spec:
-  subjects:
-  - user: "service-account-a"
-  - user: "istio-ingress-service-account"
-    properties:
-      request.auth.claims[email]: "a@foo.com"
-  roleRef:
-    kind: ServiceRole
-    name: "products-viewer"
+  {}
 {{< /text >}}
 
-In case you want to make a service publicly accessible, you can set the
-`subject` to `user: "*"`. This value assigns the `ServiceRole` to **all (both authenticated and
-unauthenticated)** users and services, for example:
+#### Custom conditions
+
+You can also use the `when` section to specify additional conditions, for example, the following
+`AuthorizationPolicy` definition includes a condition that `request.headers[version]` is either `"v1"` or `"v2"`.
+In this case, the key is `request.headers[version]`, which is an entry in the Istio attribute `request.headers`,
+which is `map` data type.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRoleBinding
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: binding-products-allusers
-  namespace: default
+ name: httpbin
+ namespace: foo
 spec:
-  subjects:
-  - user: "*"
-  roleRef:
-    kind: ServiceRole
-    name: "products-viewer"
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.headers[version]
+     values: ["v1", "v2"]
 {{< /text >}}
 
-To assign the `ServiceRole` to only **authenticated** users and services, use `source.principal: "*"`
-instead, for example:
+The supported `key` values of a condition are listed in the
+[conditions page](/docs/reference/config/authorization/conditions/).
+
+#### Authenticated and unauthenticated identity
+
+In case you want to make a workload publicly accessible, you need to leave the
```

Comment:
~In case~ If you want to make a workload publicly accessible, you need to leave the